### PR TITLE
Update this

### PR DESCRIPTION
**Pull request by temurbek-4224**
juda ko'p yangilikmas endi men ham yozib qo'ydimda 😁😁

### CODE REVIEW

**Comment by temurbek-4224** [comment]
go go

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 
 ## Acceptable Contributions
 
+-Do you want to become a contributor for all solutions of problems on www.binarysearch.com?
```
Comment (Mirzohid22) [inline CONTRIBUTING.md:4]: Bro change "for all solutions of ........ binarysearch.com?"  to  "the (awesome) repository"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+-Do you want to know all solutions on that site?
```
Comment (Mirzohid22) [inline CONTRIBUTING.md:5]: Not "all",  but "some", even a solution is enough to contribute

**Review by Mirzohid22 — COMMENTED** [review]
It reminded me the introductions of  ZTM courses! ("Do you want to be the top 10% contributor in the world? 😅")
Hurry up! bro,  I'm waiting to merge your changes

File: CONTRIBUTING.md (hunk 1 above)
Comment (temurbek-4224) [reply CONTRIBUTING.md:4]: Hop bro

File: CONTRIBUTING.md (hunk 2 above)
Comment (temurbek-4224) [reply CONTRIBUTING.md:5]: Hop 😅

**Pull request merged**